### ISSUE TITLE
scp over local context is slow. Investigate

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#73]
```
./spire -e '(upload {:dest "test-spire" :src "spire"})'
local spire|============================================================================| 100% 796.57 kB/s eta:0s      
{:attr-result {:result :ok}, :copy-result {:result :changed}, :result :changed}
```
< 800kb/sec

**Comment by retrogradeorbit** [comment]
now 170MB/sec

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.13
